device/src/async_device/lora_radio.rs

```diff
@@ -6,16 +6,19 @@ use lora_phy::mod_params::{BoardType, ChipType, RadioError};
 use lora_phy::mod_traits::RadioKind;
 use lora_phy::{DelayUs, LoRa};
 
-/// LoRa radio using the physical layer API in the external lora-phy crate
-pub struct LoRaRadio<RK, DLY>
+/// LoRa radio using the physical layer API in the external lora-phy crate.
+///
+/// The const generic P is the max power the radio may be instructed to transmit at. The const
+/// generic G is the antenna gain and board loss in dBi.
+pub struct LoRaRadio<RK, DLY, const P: u8, const G: i8 = 0>
 where
     RK: RadioKind,
     DLY: DelayUs,
 {
     pub(crate) lora: LoRa<RK, DLY>,
     rx_pkt_params: Option<lora_phy::mod_params::PacketParams>,
 }
-impl<RK, DLY> LoRaRadio<RK, DLY>
+impl<RK, DLY, const G: i8, const P: u8> LoRaRadio<RK, DLY, P, G>
```

Comment:
Here, the `G` and `P` are in different order.

Reply:
Good catch! Thank you